R/assign_peptide_type.R

```diff
@@ -54,6 +54,11 @@ assign_peptide_type <- function(data,
                                 aa_before = aa_before,
                                 last_aa = last_aa,
                                 aa_after = aa_after) {
+
+  if (missing(aa_before) | missing(last_aa) | missing(aa_after)) {
+    stop("Please provide all 3 required amino acids (aa_before, last_aa and aa_after).")
+  }
+  
```

Comment:
Unfortunately you cannot specifically ask for the arguments to be provided because then the function won't work anymore with its default arguments.
I think it was not a problem for Max since his input data frame contained a column called `aa_after` the function would recognise it and automatically use it if not provided.

What you can do for example is:

```
data <- data.frame(
  aa_before = c("K", "S", "T"),
  last_aa = c("R", "K", "Y"),
  aa_after = c("T", "R", "T")
)

assign_peptide_type(data)
```
This will run without a problem since the columns are named correctly.

If you basically request that the 3 columns are provided then you cannot use the function in its simpler version. I did this since we were requested by the reviewers to provide as many default arguments as possible when we wrote the paper.

So I would just close the pull request instead of merging it.